test/config/achievement-rule-checker-config.yml

```diff
@@ -22,6 +23,30 @@ achievements:
         requirements:
             - xp: XP
               amount: 10
+    EqualAchievementFailing:
+        requirements:
+            - xp: XP
+              amount: "== 9"
+    EqualAchievementSucceeding:
+            requirements:
+                - xp: XP
+                  amount: "== 10"
```

Comment:
The indent seems off here.

Reply:
@frederike-ramin you actually fixed a different indentation problem in https://github.com/schul-cloud/gamification/pull/50/commits/e69a1b13c2a9133f9118f8975fc3cc9c4cb5a3cb, this problem here still remains ⛵️

Reply:
Oh man..I was wondering why the comment wouldn't close. :D